React/src/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,92 @@
+import { Button, TextField } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import './style.less';
+
+type DialogTypes = {
+    dialogText: string;
+    confirmLabel: string;
+    inputsInfo: any;
+    confirmAction: any;
+    cancelAction: any;
+}
+
+const CustomTextField = styled(TextField)(({ theme }) => ({
+    border: '2px solid #96C0FB',
+    borderRadius: '8px',
+    label: { shrink: true, color: theme.palette.secondary.contrastText },
+    input: { color: theme.palette.secondary.contrastText },
+  }));
+
+const Dialog = ({ dialogText, confirmLabel, inputsInfo, confirmAction, cancelAction }: DialogTypes) => {
+    const inputsHeight = 90 * inputsInfo.length;
+    const dialogHeight = 110;
+    const totalHeight = (inputsHeight+dialogHeight)+"px";
+    
+  return (
+    <div className="blur-background">
+        <div className="dialog" style={{height: totalHeight}}>
```

Comment:
I think we should use MUI Components, not html elements. We could use for example Box or Container or Grid